extensions/amp-video-iframe/amp-video-iframe.md

```diff
@@ -114,6 +126,8 @@ function onAmpIntegrationReady(ampIntegration) {
 }
 ```
 
+**Never play the video inside the frame automatically.** Instead, you should support the integration script and use the `amp-video-iframe` tag with the `autoplay` attribute. The AMP component will autoamtically send the necessary signals to your iframe to autoplay for a better user experience.
```

Comment:
wish we could actually enforce this through a feature policy. Brought it up with Web Platform folks.